src/SprykerEco/Zed/AfterPay/Communication/Plugin/Oms/Condition/IsRefundCompletedPlugin.php

```diff
@@ -24,6 +24,8 @@ class IsRefundCompletedPlugin extends AbstractPlugin implements ConditionInterfa
     public const REFUND_TRANSACTION_ACCEPTED = AfterPayConfig::API_TRANSACTION_OUTCOME_ACCEPTED;
 
     /**
+     * {@inheritDoc}
+     *
```

Comment:
Add a specification to a method:
- Checks if the refund operation was successfully accepted for the order.